src/errors/no_renderings.html

```diff
@@ -8,6 +8,15 @@
             IMAGE is unable to render this graphic. If you would like to let us know about this, please send a link to this graphic and a short description of the problem via email: <a href = "mailto: image@cim.mcgill.ca"> image@cim.mcgill.ca</a> or report bugs using <a id="feedback-a" href="../feedback/feedback.html" target="_blank">our feedback form</a>.
```

Comment:
This paragraph should also be updated to the text in https://github.com/Shared-Reality-Lab/IMAGE-server/issues/468#issuecomment-1231930808